ci.sh

```diff
@@ -47,7 +47,7 @@ sudo mv out/linux-amd64/crc /usr/local/bin/
 popd
 
```

Comment:
I'd expand the commit log a bit, `Giving more resources to the cluster we run the testsuite on should hopefully prevent some disruptions/timeouts/... because of low memory conditions`